packages/libs/components/src/plots/NetworkPlot.tsx

```diff
@@ -76,6 +80,8 @@ function NetworkPlot(props: NetworkPlotProps, ref: Ref<HTMLDivElement>) {
     labelTruncationLength = 20,
     emptyNetworkContent,
     annotations,
+    selectedNodeLabels,
+    showNodeLabels = false,
```

Comment:
originally i thought `false` would be the best default. But after playing around with it a bit i was wrong! Let's do `true` as the default please!

Reply:
@asizemore as the toggle switch is removed, showNodeLabels prop is also removed